pkgs/os-specific/linux/hostapd/default.nix

```
@@ -1,7 +1,11 @@
 { stdenv, fetchurl, pkgconfig, libnl, openssl, sqlite ? null }:
 
 with stdenv.lib;
-stdenv.mkDerivation rec {
+let noScanPatch = fetchurl {
+      url="https://git.telliq.com/gtu/openwrt/raw/master/package/network/services/hostapd/patches/300-noscan.patch";
```

```suggestion
      url = "https://git.telliq.com/gtu/openwrt/raw/master/package/network/services/hostapd/patches/300-noscan.patch";
```

Also, that's probably not the official upstream url…